src/assets/Icons.tsx

```diff
@@ -8,7 +8,10 @@ export const KjønnAnnet = () => (
         version="1.1"
         xmlns="http://www.w3.org/2000/svg"
         focusable={false}
+        role="img"
+        aria-labelledby="kjonnAnnetIkon"
     >
+        <title id="kjonnAnnetIkon">Ukjent kjønn</title>
```

Comment:
Kan det ikke være `ø` her?

Reply:
Tja, godt mulig, men siden ID ikke vises uansett så orket jeg ikke sjekke.

Reply:
soppelbotte

Reply:
?

Reply:
Den kommer litt lenger ned her, `soppelbotteIkon`.

Reply:
Gjør ikke noe altså, jeg bare leste det og lollet litt for meg sjøl.